logo-ann/generation/01_generate_image_dump.py

```diff
@@ -148,9 +153,11 @@ def iter_jsonl(path: pathlib.Path):
 BARCODE_PATH_REGEX = re.compile(r"^(...)(...)(...)(.*)$")
 
 
-def split_barcode(barcode: str) -> List[str]:
```

Comment:
C'est bien une List[str] le type de retour (on ne renvoie jamais None)

Reply:
Actually we do now. Because of the barcode error, we return None.